component-catalog-connectors/mlx-connector/mlx_catalog_connector/mlx-catalog.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://raw.githubusercontent.com/elyra-ai/elyra/master/elyra/metadata/schemas/meta-schema.json",
+  "$id": "https://raw.githubusercontent.com/elyra-ai/examples/master/component-catalog-connectors/mlx-connector/mlx_catalog_connector/mlx-catalog.json",
+  "title": "Machine Learning Exchange Component Catalog",
+  "name": "mlx-catalog",
+  "display_name": "Machine Learning Exchange Component Catalog",
+  "schemaspace": "component-registries",
+  "schemaspace_id": "ae79159a-489d-4656-83a6-1adfbc567c70",
+  "uihints": {
```

Comment:
I reconsidered and added

```
"metadata_class_name": "elyra.pipeline.component_metadata.ComponentCatalogMetadata",
```

even though it's a no-op with the latest update to `filter_schema`.
This should make it a bit easier in case a developer decides to implement their own metadata subclass (using the MLX connector as the blueprint) and eliminates the need for others (who do not wish to do so) to rename the dummy MLX ComponentCatalogMetadata class name in multiple places.